lib/chewy/search/response.rb

```diff
@@ -33,6 +39,17 @@ def max_score
         @max_score ||= hits_root['max_score']
       end
 
+      # Response `errors` field. Returns `nil` if there is no error.
+      # @return [Array, nil]
```

Comment:
Would be nice to have as `sig` :)

Reply:
Didn't find any sorbet reference in Chewy repo, hence didn't use it.